services/provider/proto/provider.proto

```diff
@@ -118,13 +118,26 @@ message StorageConfigResponse{
     SystemAttributes systemAttributes = 3;
 }
 
+message RbdDriverRequirements {
+  bool omapInfo = 1;
```

Comment:
@nb-ohad, do we want to move this to operatorConfig Spec instead of Driver Spec and always deploy it?